sampleconfig/configtx.yaml

```diff
@@ -72,6 +72,31 @@ Organizations:
             - Host: 127.0.0.1
               Port: 7051
 
+        # PrivateDataImplicitCollection stores the configuration for this org's implicit
+        # collection.
+        PrivateDataImplicitCollection:
+            # MaxPeerCount is the maximum number of other peers (across authorized organizations)
```

Comment:
I would put RequiredPeerCount before MaxPeerCount. That's how it is in the associated collection config doc, and it is most natural to think of dissemination for "0 to 1" peers rather than dissemination for "1 to 0" peers.

Reply:
Sounds good, I'm not sure why I had MaxPeerCount first, but I distinctly remember doing so because I saw somewhere it was before Required. I can't find where now so maybe I was just seeing things. I'll swap the orders where necessary.